bundle/rhpam/manifests/rhpam-kogito-operator-controller-manager-metrics-service_v1_service.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  creationTimestamp: null
```

Comment:
can be removed.

Reply:
its for accessing operator on secure `https` port.

Reply:
comes by default with operator-sdk

Reply:
I meant, creationTimestamp is not needed., it is added anyways later

Reply:
done